.gitignore

```diff
@@ -1 +1,23 @@
-.vscode
\ No newline at end of file
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
**REQUIRED**
* Let's delete all unnecessary files and assets. 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))